app/views/shared/_share_buttons.html.erb

```diff
@@ -1,18 +0,0 @@
-<div class="grid-row sidebar-with-body dont-print">
```

Comment:
We could keep this partial to avoid repetition of the grid classes in the content items. Maybe rename to something like `share_links_offset_in_grid` and then include the component call?